tests/unit/test_game.py

```diff
@@ -469,3 +409,13 @@ def test_fictitious_play(self, A, B, seed):
             assert np.array_equal(row_play, expected_row_play)
             assert np.array_equal(column_play, expected_column_play)
         # assert expected_outcome == outcome
+
+    def test_replicator_dynamics(self):
+        """Test for the replicator dynamics algorithm"""
+        A = np.array([[3, 2], [4, 1]])
+        game = nash.Game(A)
+        y0 = np.array([0.9, 0.1])
+        timepoints = np.linspace(0, 10, 100)
+        xs = game.replicator_dynamics(y0, timepoints)
+        expected_xs = np.array([[0.50449178, 0.49550822]])
+        assert np.allclose(xs[-1], expected_xs)
```

Comment:
Can we add a second test for a larger game (perhaps with a 5 by 5 matrix?).

Reply:
Will do.